### PR TITLE
issue#35 endpoint update annotation #35

### PR DESCRIPTION
**Pull request by berheg**
#35

### CODE REVIEW

File: src/server/api/routes/annotations.router.js [hunk 1]
```diff
@@ -34,4 +34,11 @@ router.get('/:id', (req, res, next) => {
     .then((result) => res.json(result))
     .catch(next);
 });
+// ENDPOINT: /api/annotations/ :PATCH
```
Comment (OlkaA) [inline src/server/api/routes/annotations.router.js]: Very minor thing.
I think it is better to comment endpoint like this:
// ENDPOINT: /api/annotations/:id :PATCH

**Comment by OlkaA** [comment]
Endpoint works good. Annotation is updating.
Swagger also works properly with this endpoint.

**Review by OlkaA — APPROVED** [review]

**Review by mo3ni — APPROVED** [review]
well done 👌

File: src/server/api/routes/annotations.router.js (hunk 1 above)
Comment (bytewiz) [reply src/server/api/routes/annotations.router.js]: ☝️ Agree

**Review by bytewiz — CHANGES_REQUESTED** [review]
Please check the comment made from Olha and add that one very small thing then it will be approved 👍 🥇 🔥

**Review by bytewiz — APPROVED** [review]
Looks good to me 👍

**Pull request merged**